plugins/haptics/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tauri-plugin-haptics"
-version = "1.0.0"
+version = "2.0.0-beta.1"
```

Comment:
```suggestion
version = "2.0.0-beta.0"
```
covector may not like that either but i forgot how that works for initial prereleases but we'll see 🤷